mobile/src/screens/chat/components/badge/badge.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { styles } from './styles';
+
+const Badge: React.FC = () => {
+  return (
+    <View style={styles.badge}>
+      {/*12 is moked data*/}
+      <Text style={styles.badgeText}>12</Text>
```

Comment:
pass this value via props, and mock it on the top level
```suggestion
      <Text style={styles.badgeText}>{label}</Text>
```

Reply:
Done